### PR TITLE
update docs for links to work

### PR DESCRIPTION
**Pull request by tim-allen-ck**
# Resolves HASH_SIGN_FOLLOWED_BY_ISSUE_NUMBER

## What is being addressed

Current Links for case study docs do not work. This should fix that, and open them in a new tab.

### CODE REVIEW

File: docs/azure-tre-overview/case-studies.md [hunk 1]
```diff
@@ -4,17 +4,17 @@
...
 
-Click [here](https://cloudkubed.com/case-studies/imperial-college-london-trusted-research-environment/ "Imperial College London TRE") to see the full case study.
+Click [here](https://cloudkubed.com/case-studies/imperial-college-london-trusted-research-environment/ "CloudKubed and Imperial College London"){target="_blank"} to see the full case study.
 
 ## Building the Trusted Research Environment with Azure Databricks
 The importance of secure and scalable data environments in healthcare research has grown, particularly highlighted by the COVID-19 pandemic. The Trusted Research Environments (TREs) enable secure collaboration on sensitive data while maintaining patient privacy. Azure Databricks enhances TREs by offering scalable, secure, and flexible data processing capabilities.
 For example, Providence Health used Azure Databricks to reduce hospital overcrowding and improve patient care by leveraging real-time data. The integration of Azure Databricks into the Azure TRE blueprint allows researchers to perform complex data analysis securely, promoting innovative research and improved healthcare outcomes.
```
Comment (marrobi) [inline docs/azure-tre-overview/case-studies.md:11]: Can you remove the reference to Providence Health as not sure they actually use Azure TRE, just use Databricks .

Comment (tim-allen-ck) [reply docs/azure-tre-overview/case-studies.md:11]: is this better?

Comment (marrobi) [reply docs/azure-tre-overview/case-studies.md]: thanks.

**Review by marrobi — APPROVED** [review]

**Review by Danny-Cooke-CK — APPROVED** [review]

**Pull request merged**